### PR TITLE
Video logger

### PR DESCRIPTION
**Pull request by ablattmann**

### CODE REVIEW

**Comment by ablattmann** [comment]
Add video logger to sbord. Requires ffmpeg to be installed on the executing system.

File: sbord/sbord.py [hunk 1]
```diff
@@ -13,13 +13,15 @@
 import plotly.graph_objects as go
 from natsort import natsorted
 from scipy.signal import savgol_filter
+import subprocess
+import cv2
```
Comment (pesser) [inline sbord/sbord.py:17]: Not needed, is it? Could you delete it to reduce dependencies?

**Review by pesser — CHANGES_REQUESTED** [review]
Cool thank you! LGTM except for the unnecessary cv2 import

File: sbord/sbord.py (hunk 1 above)
Comment (ablattmann) [reply sbord/sbord.py]: sure, just a remnant I forgot to delete :)

Comment (ablattmann) [reply sbord/sbord.py]: should be removed by now

**Review by pesser — APPROVED** [review]

**Pull request merged**